executor/aggregate_test.go

```diff
@@ -307,10 +308,12 @@ func (s *testSuite) TestAggregation(c *C) {
 
 func (s *testSuite) TestStreamAgg(c *C) {
 	col := &expression.Column{
-		Index: 1,
+		Index:   1,
+		RetType: types.NewFieldType(mysql.TypeUnspecified),
```

Comment:
`TypeUnspecified` should be avoided,  use `TypeLonglong` instead, since the input value is interger.